packages/analytics/src/index.ts

```diff
@@ -1,11 +1,11 @@
 import mixpanel, { Mixpanel } from 'mixpanel';
 
 export class Analytics {
-  protected project: string;
+  protected meta: any;
```

Comment:
What I meant by `meta` is for each event. Each event should have a `meta` field.